packages/web3-core-requestmanager/src/index.js

```diff
@@ -20,7 +20,7 @@
  * @date 2017
  */
 
-"use strict";
```

Comment:
This file looks like it can be removed the change-set. Looks like unrelated lints...

Reply:
I was intentionally doing these code style improvements.